### ISSUE TITLE
LibreOffice installation not found under OS X 10.9

### ISSUE THREAD

**Issue by AlexanderWillner** [unoconv/unoconv#177]
Description: LibreOffice installation not found under OS X 10.9
Version LibreOffice: 4.1.4.2
URL LibreOffice:http://www.libreoffice.org
Version unoconv: 0.6
URL unoconv: http://dag.wieers.com/home-made/unoconv/unoconv-0.6.tar.gz 

Steps that will reproduce the problem?
1. Download and install LibreOffice (version and URL above)
2. Download and extract unoconv (version and URL above)
3. Run "./unoconv"

What is the expected result?
To have a running unoconv installation

What happens instead?

```
$ ./unoconv 
unoconv: Cannot find a suitable office installation on your system.
ERROR: Please locate your office installation and send your feedback to:
       http://github.com/dagwieers/unoconv/issues
```

Possible workaround:
Unknown

Any additional information:

```
 python /Applications/LibreOffice.app/Contents/MacOS/uno.py 
Traceback (most recent call last):
  File "/Applications/LibreOffice.app/Contents/MacOS/uno.py", line 21, in <module>
    import pyuno
ImportError: dlopen(/Applications/LibreOffice.app/Contents/MacOS/pyuno.so, 2): no suitable image found.  Did find:
    /Applications/LibreOffice.app/Contents/MacOS/pyuno.so: mach-o, but wrong architecture
```

**Comment by regebro** [comment]
All our integration tests fail on Mac OS lately, something has happened there. I don't have OS X available, and I don't personally need OS X support, so I'm looking for help here, somebody that are willing to try to debug this and figure out what fails.

**Comment by munichman2008** [comment]
I found out you can instead use the latest version of LibreOffice and simply convert your documents using the command-line options. It seems to me an easier and more straight-forward way of converting documents. Follow the instructions here: https://ask.libreoffice.org/en/question/12084/how-to-convert-documents-to-pdf-on-osx/

**Comment by regebro** [comment]
Yes, but that's not practical if you convert a lot of documents, as you will have to start Libreoffice every time you convert.

But absolutely, if you only convert a document here and there, that's probably a better solution.
